draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -309,6 +316,13 @@ Statistics related to ICE candidates that are accessible to the web
 application MUST NOT contain the IP address of a local or remote mDNS
 candidate; the mDNS name SHOULD be used instead.
 
+Statistics SHOULD NOT leak whether the mDNS resolution succeeds or fails.
+For that reason, RTCIceCandidateStats objects as defined in {{WebRTCStats}}
+SHOULD be generated for any remote mDNS candidate submitted to the ICE agent,
+even if the mDNS candidate is ignored as part of {{processing}}.
+An implementation strategy to obfuscate the address of an mDNS candidate in the statistics, regardless if it is resolved or not, is to replace
+the mDNS hostname of the ICE candidate with IP values "0.0.0.0" or "::".
```

Comment:
Not sure I understand this guidance. Wouldn't we just want stats.address to contain the mDNS name?